extensions/google-cloud-functions/deployment/src/main/java/io/quarkus/gcp/functions/deployment/CloudFunctionDeploymentBuildStep.java

```diff
@@ -21,6 +23,12 @@ public UberJarRequiredBuildItem forceUberJar() {
         return new UberJarRequiredBuildItem();
     }
 
+    @BuildStep(onlyIf = NativeSourcesBuild.class)
+    void failForNativeSources(BuildProducer<ArtifactResultBuildItem> artifactResultProducer) {
+        throw new IllegalArgumentException(
+                "The Goog Cloud extensions are incompatible with the 'native-sources' package type.");
```

Comment:
Typo `Google`

Reply:
The surprising thing would be for a PR of mine to **not** contain a typo 😆

Reply:
Fixed